error.go

```diff
@@ -129,7 +129,7 @@ type ActiveEndpointsError struct {
 }
 
 func (aee *ActiveEndpointsError) Error() string {
-	return fmt.Sprintf("network %s has active endpoints", aee.name)
+	return fmt.Sprintf("network %s id %s has active endpoints", aee.name, aee.id[0:12])
```

Comment:
@fcrisciani 
I don‘t know whether the user is using xxx is id or name , I think print name and id is better.
`docker network ls`  result `NETWORK ID`  length is 12, so here print  12.